flowredux/src/commonMain/kotlin/com/freeletics/flowredux/dsl/InStateBuilderBlock.kt

```diff
@@ -139,7 +139,7 @@ public class InStateBuilderBlock<InputState : S, S : Any, A : Any>(
     ) {
         onEnter { state ->
             handler(state.snapshot)
-            InternalNoStateChange
+            NoStateChange
```

Comment:
No need to change anything but I am just wondering if we should call `state.noChange()` instead of `NoStateChange´ directly to only have in the future one single point that we need to change ... but could also fire back (i.e. if we do intend only to make "internal" changes), so I would probably keep it as you have it. So everything is fine ... I was just  thinking out loud ...